keyboards/planck/ez/config.h

```diff
@@ -140,6 +140,11 @@
 #define RGB_MATRIX_KEYPRESSES
 #define RGB_MATRIX_FRAMEBUFFER_EFFECTS
 
+#define RGB_MATRIX_LED_PROCESS_LIMIT 5
+#define RGB_MATRIX_LED_FLUSH_LIMIT 26
+
+#define I2C1_CLOCK_SPEED 400000
```

Comment:
This wont do anything as its a I2Cv1 setting and planck ez is f303 which is I2Cv2.
```suggestion
```

Reply:
Okay, good to know